ci-operator/step-registry/netobserv/perf-test/deploy/netobserv-perf-test-deploy-commands.sh

```diff
@@ -17,7 +17,18 @@ source scripts/netobserv.sh
 deploy_lokistack
 deploy_kafka
 deploy_netobserv
-createFlowCollector "-p KafkaConsumerReplicas=${KAFKA_CONSUMER_REPLICAS}"
+
+PARAMETERS="-p KafkaConsumerReplicas=${KAFKA_CONSUMER_REPLICAS}"
+
+if [[ -n $MULTISTAGE_PARAM_OVERRIDE_SAMPLING ]]; then
+    PARAMETERS="${PARAMETERS} EBPFSamplingRate=${MULTISTAGE_PARAM_OVERRIDE_SAMPLING}"
+fi
+
+if [[ -n $MULTISTAGE_PARAM_OVERRIDE_LOKI_ENABLE ]]; then
+    PARAMETERS="${PARAMETERS} LokiEnable=${MULTISTAGE_PARAM_OVERRIDE_LOKI_ENABLE}"
+fi
+
+createFlowCollector ${PARAMETERS}
```

Comment:
to make sure variable expansions are happening well here in bash, we should run a rehearsal of NDH with `MULTISTAGE_PARAM_OVERRIDE_SAMPLING` set

Reply:
can we run this pre-merge?

Reply:
@memodi I have overriden sampling and also commented out `upload_baseline`; since if the numbers qualify to be baseline then they will be uploaded since the if condition passes as they are not being triggered by gangway

Reply:
ah okay, will uncomment it then. Thanks for the info!